### ISSUE TITLE
Master branch in 218 doesn't compile without local libraries

### ISSUE THREAD

**Issue by jbeorse** [opendatakit/opendatakit#1368]
#### Software and hardware versions 
Survey and Tables rev 218/219

#### Problem description
The master branch of rev218/219 will not compile without androidlibrary and androidtables checked out locally.

#### Steps to reproduce the problem
Checkout Tables locally. Check out latest (commit  2de32dc). Run ./gradle clean assemble. Get error:

```
> Task :tables_app:compileDemoBasicDebugJavaWithJavac
/home/jbeorse/workspace/odk-android/tables/tables_app/src/main/java/org/opendatakit/tables/views/webkits/OdkTablesWebView.java:60: error: method does not override or implement a method from a supertype
   @Override public void loadPage() {
   ^
/home/jbeorse/workspace/odk-android/tables/tables_app/src/main/java/org/opendatakit/tables/views/webkits/OdkTablesWebView.java:62: error: cannot find symbol
      boolean pageFrameworkLoaded = hasPageFrameworkFinishedLoading();
                                    ^
  symbol:   method hasPageFrameworkFinishedLoading()
  location: class OdkTablesWebView
/home/jbeorse/workspace/odk-android/tables/tables_app/src/main/java/org/opendatakit/tables/views/webkits/OdkTablesWebView.java:68: error: cannot find symbol
         loadPageOnUiThread(baseUrl, getContainerFragmentID(), LoadType.FULL_LOAD);
                                                               ^
  symbol:   variable LoadType
  location: class OdkTablesWebView
/home/jbeorse/workspace/odk-android/tables/tables_app/src/main/java/org/opendatakit/tables/views/webkits/OdkTablesWebView.java:91: error: cannot find symbol
         loadPageOnUiThread(baseUrl, getContainerFragmentID(), LoadType.RELOAD);
```

#### Expected behavior
Compiles normally. Which does work if androidlibrary and androidcommon are checked out locally.

**Comment by wbrunette** [comment]
I believe this was fixed with this commit https://github.com/opendatakit/tables/commit/064f92f6b01ff405447ccf826e9960de2fb80f6a

fixed opendatakit/tables#54 which hasn't been merged to master yet

**Comment by wbrunette** [comment]
Additionally the correct command is ./gradle clean assembleMasterBasic, assemble will try to build all 3 variants
